File: python/python3/Java/Python3ParserBase.java
import org.antlr.v4.runtime.*;

public abstract class Python3ParserBase extends Parser
{
    protected Python3ParserBase(TokenStream input)
    {
	super(input);
    }

    public boolean CannotBePlusMinus()
    {
	return true;
    }

    public boolean CannotBeDotLpEq()
    {
	return true;
    }
}

File: python/python3/Dart/Python3ParserBase.dart
import 'package:antlr4/antlr4.dart';
import 'dart:io';
import 'dart:convert';

abstract class Python3ParserBase extends Parser
{
    Python3ParserBase(TokenStream input) : super(input)
    {
    }

    bool CannotBePlusMinus()
    {
        return true;
    }

    bool CannotBeDotLpEq()
    {
        return true;
    }
}
